test/unit_test/instantiate_fft_tests.hpp

```diff
@@ -99,6 +101,12 @@ INSTANTIATE_TEST_SUITE_P(SubgroupTest, FFTTest,
                              all_valid_placement_layouts, fwd_only, complex_storages, ::testing::Values(1, 3, 555),
                              ::testing::Values(sizes_t{64}, sizes_t{96}, sizes_t{128}))),
                          test_params_print());
+// sizes that use subgroup implementation
+INSTANTIATE_TEST_SUITE_P(SubgroupRegressionTest, FFTTest,
+                         ::testing::ConvertGenerator<basic_param_tuple>(
+                             ::testing::Combine(ip_batch_interleaved_layout, fwd_only, interleaved_storage,
+                                                ::testing::Values(44, 100), ::testing::Values(sizes_t{100}))),
```

Comment:
IIRC size 80 was hanging as well with any batch size. could you add it here?
If it so turns out to be a different bug, we would shift that to a different PR

Reply:
Done in [84aa2cf](https://github.com/codeplaysoftware/portFFT/pull/130/commits/84aa2cf368a8ccba8d036cbc89e60dec8590e775). This did not cause additional hanging.